<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Map3DExportWidget</class>
 <widget class="QWidget" name="Map3DExportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>251</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="exportFormatLabel">
       <property name="text">
        <string>Export format</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QComboBox" name="exportFormatComboxBox"/>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QgsSpinBox" name="terrainTextureResolutionSpinBox">
       <property name="minimum">
        <number>16</number>
       </property>
       <property name="maximum">
        <number>4096</number>
       </property>
       <property name="value">
        <number>512</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QCheckBox" name="smoothEdgesCheckBox">
       <property name="text">
        <string>Smooth edges</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="folderLAbel">
       <property name="text">
        <string>Folder</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Terrain texture resolution</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="sceneNameLabel">
       <property name="text">
        <string>Scene name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QgsFileWidget" name="selectFolderWidget" native="true"/>
     </item>
     <item row="9" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0" colspan="3">
      <widget class="QCheckBox" name="exportTexturesCheckBox">
       <property name="text">
        <string>Export textures</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QgsSpinBox" name="terrainResolutionSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>4096</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QCheckBox" name="exportNormalsCheckBox">
       <property name="text">
        <string>Export normals</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Terrain resolution</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="sceneNameLineEdit"/>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QgsDoubleSpinBox" name="scaleSpinBox">
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Model scale</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sceneNameLineEdit</tabstop>
  <tabstop>terrainResolutionSpinBox</tabstop>
  <tabstop>terrainTextureResolutionSpinBox</tabstop>
  <tabstop>scaleSpinBox</tabstop>
  <tabstop>smoothEdgesCheckBox</tabstop>
  <tabstop>exportNormalsCheckBox</tabstop>
  <tabstop>exportTexturesCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
